### PR TITLE
✨ Add Buckwheat

### PR DESCRIPTION
**Pull request by shrinerh**

### CODE REVIEW

File: src/foxcraft_dungeon_loot/items/love_wheat.mc [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import ../../macros.mcm
+
+function clock_3s {
+    schedule function foxcraft_dungeon_loot:items/love_wheat/clock_3s 3s
+
+    execute as @a at @s if score @s satyrn.fdl.itemId.mainHand matches 76 run {
+        # Up to 10 livestock mobs within 5 blocks with an age of 0 (breeding cooldown is over)
+        execute if entity @e[type=#foxcraft_dungeon_loot:livestock, distance=..5, nbt={Age: 0}, limit=10, sort=nearest] as @e[type=#foxcraft_dungeon_loot:livestock, distance=..5, nbt={Age: 0}, limit=10, sort=nearest] at @s run {
```
Comment (satyrnidae) [inline src/foxcraft_dungeon_loot/items/love_wheat.mc]: Replace NBT check w/ predicate check

**Review by satyrnidae — CHANGES_REQUESTED** [review]
cool just make a predicate w/ entity properties

File: src/foxcraft_dungeon_loot/items/love_wheat.mc (hunk 1 above)
Comment (shrinerh) [reply src/foxcraft_dungeon_loot/items/love_wheat.mc]: done

**Review by satyrnidae — APPROVED** [review]
nice

**Pull request merged**